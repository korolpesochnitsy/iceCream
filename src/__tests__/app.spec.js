jest.mock('../structure/Header', () => () => <span>Header</span>);
jest.mock('../structure/Footer', () => () => <span>Footer</span>);

import React from 'react';
import { render, cleanup } from '@testing-library/react';
import App from '../App';

describe('App', () => {
  afterEach(cleanup);

  it('should render the app root', () => {
    const { container } = render(<App />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
