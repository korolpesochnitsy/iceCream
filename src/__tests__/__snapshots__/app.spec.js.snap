// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the app root 1`] = `
<div
  data-testid="router"
>
  <a
    class="skip-link"
    href="#main"
  >
    Skip to content
  </a>
  <span>
    Header
  </span>
  <div
    data-testid="switch"
  >
    <span>
      Menu
    </span>
    <span>
      /
    </span>
    <span>
      IceCreams
    </span>
    <span>
      /ice-creams
    </span>
    <span>
      AddIceCream
    </span>
    <span>
      /menu-items/add
    </span>
    <span>
      EditIceCream
    </span>
    <span>
      /menu-items/:menuItemId
    </span>
    <span>
      Redirect to 
      /
    </span>
  </div>
  <span>
    Footer
  </span>
</div>
`;
